server/database/models/coffee.js

```diff
@@ -0,0 +1,18 @@
+const Sequelize = require('sequelize');
+const sequelize = require('./sequelize.js');
+
+const Coffee = sequelize.define('coffee', {
+  price: {
+    type: Sequelize.FLOAT,
+  },
+  note: {
+    type: Sequelize.STRING(500),
+  },
+  time: {
+    type: Sequelize.DATE,
+  },
+  event_id: {
```

Comment:
you don't have to add the foreign key here, because you identified it inside ```index.js``` file  like this```Coffee.belongsTo(Event, {
  onDelete: 'CASCADE', forignKey: 'event_id', targetKey: 'id',
});```

Reply:
done :+1: